utils/global-state-update-gen/src/validators.rs

```diff
@@ -1,27 +1,60 @@
 mod validators_manager;
 
+use std::collections::BTreeMap;
+
 use clap::ArgMatches;
 
+use casper_types::{AsymmetricType, PublicKey, U512};
+
 use validators_manager::ValidatorsUpdateManager;
 
+pub struct ValidatorConfig {
+    stake: U512,
+    maybe_balance: Option<U512>,
```

Comment:
Is this the _new_ balance? Maybe that would be a better name. Otherwise it looks like a validator could be created but his purse not.